### PR TITLE
chore(framework, create-novu-app, novu): Publish prerelease

### PR DESCRIPTION
**Pull request by rifont**
### What changed? Why was the change needed?
* Publish `@novu/framework`
* Publish `create-novu-app`
* Publish `novu`

### Screenshots

<details>
<summary><strong>Expand for optional sections</strong></summary>

### Special notes for your reviewer

</details>

### CODE REVIEW

File: packages/create-novu-app/templates/app-react-email/ts/app/api/novu/route.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { myWorkflow } from "../../novu/workflows";
 
 const client = new Client({ strictAuthentication: false });
 
-export const { GET, POST, PUT, OPTIONS } = serve({
```
Comment (rifont) [inline packages/create-novu-app/templates/app-react-email/ts/app/api/novu/route.ts:7]: `PUT` isn't needed by Novu framework.

File: packages/framework/README.md
Comment (rifont) [inline packages/framework/README.md:1]: Another pass at the README.md to reflect latest changes. We can refine later, this is just to get us 90% of the way there.

File: packages/framework/package.json [hunk 2]
```diff
@@ -104,7 +104,7 @@
     "zod-to-json-schema": "^3.23.0"
   },
   "dependencies": {
-    "@novu/shared": "0.24.2",
+    "@novu/shared": "workspace:*",
```
Comment (rifont) [inline packages/framework/package.json:107]: Should be checked-in as `workspace:*`, but modified prior to publish.

File: pnpm-lock.yaml
Comment (rifont) [inline pnpm-lock.yaml:1]: Changes due to using the `workspace:*` directive.

File: packages/framework/package.json (hunk 2 above)
Comment (scopsy) [reply packages/framework/package.json:107]: How is itmodified?

**Review by scopsy — APPROVED** [review]

**Review by scopsy — APPROVED** [review]

File: packages/framework/package.json (hunk 2 above)
Comment (rifont) [reply packages/framework/package.json:107]: Manually for now, but `pnpm version` has a capability to replace it, publish, then revert it automatically

**Pull request merged**